### PR TITLE
[CUR-561] Added cover page to docx generation

### PR DESCRIPTION
**Pull request by orangemug**
## Description
Added finalised version of cover page to docx file generation 

## Issue(s)

`CUR-561`

## How to test

1. Go to https://deploy-preview-2429--oak-web-application.netlify.thenational.academy/teachers/curriculum/docx-poc/select
2. Choose a subject via the autoComplete
3. Download the docx template - linked on the page
4. Download the rendered document
5. Check the cover page is present and correct

## Screenshots

Screenshot 2024-05-17 at 11 52 08

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner
- [ ] Does this PR update a package with a breaking change

### CODE REVIEW

**Comment by rightsaidjames** [comment]
@orangemug Some questions/issues. I expect some of these are issues with the template, not the code.

- For secondary subjects (other than Maths), should the KS4 exam board be displayed on the cover page?
- The alt text for the Oak logo on the cover page should be `Oak National Academy` - the word 'logo' isn't needed because that's not what a screen reader user needs to know in this context.
- The subject illustration on the cover page isn't changing per subject. It should also be marked as decorative.
- Whenever I open a generated doc I get this message, and I have to choose Yes in order to be able to view the document:
Screenshot 2024-05-20 at 11 12 37

Here are my generated files:
[History - Secondary - AQA - 20-05-2024 12-49-08.docx](https://github.com/oaknational/Oak-Web-Application/files/15376693/History.-.Secondary.-.AQA.-.20-05-2024.12-49-08.docx)
[Science - Primary - 20-05-2024 11-12-16.docx](https://github.com/oaknational/Oak-Web-Application/files/15376694/Science.-.Primary.-.20-05-2024.11-12-16.docx)

**Comment by orangemug** [comment]
Notes:

 - Accessibility issues - we'll deal with those after phase 1 of the docx
 - I diagnosed with James and it looks like something is up with the ms word install. No other machine gets this error message.

File: src/pages-helpers/curriculum/docx/curriculum-download-patcher/patches/cover.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import type { Element } from "xml-js";
+
+import { CombinedCurriculumData } from "..";
+
+import { textIncludes, textReplacer } from "./util";
+
+function keyStageFromPhaseTitle(phaseTitle: string) {
```
Comment (sonalisin) [inline src/pages-helpers/curriculum/docx/curriculum-download-patcher/patches/cover.ts:7]: the only time where I think I would use a `case` and `switch` just for a bit more clarity, but that is being really nitpicky

**Review by sonalisin — APPROVED** [review]
Just a tiny comment re: using cases instead of an if - but happy to leave as is if it's more consistent without a switch or case

**Comment by rightsaidjames** [comment]
@orangemug Outstanding issues:
* The exam board is output as all-caps, but this doesn't work for Edexcel (`EDEXCEL`) because it's not an acronym.
* The subject illustration is still hardcoded to the English one - it should vary per subject (Maths is an abacus, History is a crown etc.)

Sample docs:
[History - Secondary - AQA - 22-05-2024 16-12-55.docx](https://github.com/oaknational/Oak-Web-Application/files/15405409/History.-.Secondary.-.AQA.-.22-05-2024.16-12-55.docx)
[English - Secondary - EDEXCEL - 22-05-2024 16-11-58.docx](https://github.com/oaknational/Oak-Web-Application/files/15405410/English.-.Secondary.-.EDEXCEL.-.22-05-2024.16-11-58.docx)

**Comment by orangemug** [comment]
> The exam board is output as all-caps, but this doesn't work for Edexcel (EDEXCEL) because it's not an acronym.

Fixed in https://github.com/oaknational/Oak-Web-Application/pull/2429/commits/807bf10b92bc721a558cfedbd84ff1ea80cc5135

**Comment by orangemug** [comment]
I'll fix missing tests in the main branch, merging!

**Pull request merged**